vms/event/services.py

```diff
@@ -1,18 +1,30 @@
 from django.core.exceptions import ObjectDoesNotExist
 from event.models import Event
 
+#check if the event exists and is not empty
+def event_not_empty(event_id):
+    result = True
+    event = get_event_by_id(event_id)
+    if not event:
+        result = False
+    return result
+
 #need to check that this event is not accociated with any jobs, 
```

Comment:
@vubo Just the comments please :) As an FYI, it's probably a good idea to use the Django 1.8 docs https://docs.djangoproject.com/en/1.8/ as the primary resource.

For example, 
```
def event_not_empty(event_id):
    """ Returns true if an event exists and is not empty """
```

We may wish to revisit the naming of these functions at a later date to improve understanding simply from the function names and it may decrease the number of `not` statements and double negation. For now, don't worry about it. I'll need to have some additional time to think on this.

Thanks for researching the issue and asking for clarification :)